libcontainer/container_linux.go

```diff
@@ -1111,7 +1111,7 @@ func (c *linuxContainer) Checkpoint(criuOpts *CriuOpts) error {
 			return err
 		}
 
-		err = ioutil.WriteFile(filepath.Join(criuOpts.ImagesDirectory, descriptorsFilename), fdsJSON, 0655)
+		err = ioutil.WriteFile(filepath.Join(criuOpts.ImagesDirectory, descriptorsFilename), fdsJSON, 0600)
```

Comment:
should we do the same for  os.Mkdir(criuOpts.WorkDirectory, 0755)?

Reply:
Sounds like a good idea. No checkpoint files should be readable by anyone else but the user creating it.